### ISSUE TITLE
Expose the cli config parser

### ISSUE THREAD

**Issue by w33ble** [egoist/bili#82]
In `cli.js` currently, it both sets up the `cac` config and calls parse. It would be handy if you exposed the config in another file, so it could be included (and extended) to build custom tooling with bili as the base.

**Comment by egoist** [comment]
why do you want to use the cli parser instead of building your own with Bili's Node.js API?

**Comment by egoist** [comment]
oh I got it after reading https://github.com/egoist/bili/issues/83

**Comment by w33ble** [comment]
Yeah, otherwise I’d end up just copying and pasting what’s here already, instead of simply adding new options to what’s already there.
